test/org/sagebionetworks/bridge/dynamodb/DynamoSchedulePlanDaoTest.java

```diff
@@ -41,7 +41,7 @@
     
     private StudyIdentifier studyIdentifier;
     
-    private Set<String> plansToDelete;
+    private Set<SchedulePlan> plansToDelete;
```

Comment:
This is a bad idea. SchedulePlans are mutable, so the HashSet won't work as expected. Furthermore, if we ever introduce a test that changes the schedule plan guid, this test will break in mysterious ways. It's much much much better to use a Set of String of guids.